flight/Libraries/math/misc_math.c

```diff
@@ -255,6 +255,43 @@ void cubic_deadband_setup(float w, float b, float *m, float *r)
 	*r = *m * powf(w, 3) + b * w;
 }
 
+/**
+ * Perform a linear interpolation over N points
```

Comment:
@kubark42 There are no constraints on the inputs or outputs; you can set the input range to -50,50 and set the curve points to [-5,-2.5,-,2.5,5] or whatever you want; the min/max output will be defined by the curve points. The [0,1] thing is only used to select the curve point index to use for interpolation, but the point at that index has no constraints on its range.